examples/material/create_material.py

```diff
@@ -1,28 +1,58 @@
 #!/usr/bin/env python
 # coding: utf-8
 
+# [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Exabyte-io/exabyte-api-examples/blob/feature/SOF-4400-skinny-req/examples/material/create_material.ipynb)
+
 # # Overview
 # 
 # In this example we create a material from a JSON config with [tags](https://docs.exabyte.io/entities-general/data/#tags) to identify the material.
 
-# # Execution
-# 
-# > <span style="color: orange">**NOTE**</span>: In order to run this example, an active Exabyte.io account is required. RESTful API credentials shall be updated in [settings](../settings.py). The generation of the credentials is also explained therein.
+# # Complete Authorization Form and Initialize Settings
 # 
-# ## Import packages
+# This will also determine environment and set all environment variables. We determine if we are using Jupyter Notebooks or Google Colab to run this tutorial. If google drive, we will have some extra steps to do.
+
+# In[ ]:
+
+
+#@title Authorization Form
+ACCOUNT_ID = "" #@param {type:"string"}
+AUTH_TOKEN = "" #@param {type:"string"}
+MATERIALS_PROJECT_API_KEY = "" #@param {type:"string"}
+ORGANIZATION_ID = ""#@param {type:"string"}
 
-# In[]:
+import os, glob
 
+if 'google.colab' in str(get_ipython()):
+  from google.colab import _message
+  notebook_name = _message.blocking_request('get_ipynb')['ipynb']['metadata']['colab']['name']
+  get_ipython().system('git clone -b feature/SOF-4400-skinny-req https://github.com/Exabyte-io/exabyte-api-examples.git    ')
+  notebook_path = glob.glob('**/'+notebook_name, recursive=True)[0][0:-len(notebook_name)]
+  os.chdir(notebook_path)
+  get_ipython().system('pip install --no-deps -r ../../requirements-colab.txt')
+  os.environ['notebook_environment'] = "Colab"
+elif 'ZMQInteractiveShell'  in str(get_ipython()):
+  os.environ['notebook_environment'] = "Jupyter"
+else:
+  os.environ['notebook_environment'] = ""
 
-import os
-import sys
 
-# Import settings and utils file
+# # Imports
+
+# In[ ]:
+
+
+import sys, importlib
 module_path = os.path.abspath(os.path.join('..'))
 if module_path not in sys.path: sys.path.append(module_path)
+from utils.generic import display_JSON, update_settings_with_users_authorization_info
+update_settings_with_users_authorization_info(ACCOUNT_ID, AUTH_TOKEN, MATERIALS_PROJECT_API_KEY, ORGANIZATION_ID)
+
+import settings; importlib.reload(settings)
 from settings import ENDPOINT_ARGS
-from utils.generic import ensure_packages_are_installed, display_JSON
-ensure_packages_are_installed()
+
+if os.environ['notebook_environment'] == "Jupyter":
```

Comment:
As we discussed - let's adjust the `ensure_packages_are_installed` functional itself to avoid having to modify other notebooks